### PR TITLE
Fix incompatible mitt library version is compatible with vue version

### PR DESCRIPTION
**Pull request by Jzow**
Modifying two files that are incompatible with the `vue` version and result in syntax errors

### CODE REVIEW

**Comment by Jzow** [comment]
CC #106 

Github CI/CD run error: 
```
Error: Dependencies lock file is not found in XX 
Supported file patterns: package-lock.json,npm-shrinkwrap.json,yarn.lock
```

It is necessary to discuss whether to add a `package-lock` file.

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
```
Comment (Jzow) [inline .github/workflows/node.js.yml:19]: It is not necessary to check version 14 in Ubnutu

**Comment by suyuan32** [comment]
Your workflow doesn't seem to be working.

**Comment by suyuan32** [comment]
> CC #106
> 
> Github CI/CD run error:
> 
> ```
> Error: Dependencies lock file is not found in XX 
> Supported file patterns: package-lock.json,npm-shrinkwrap.json,yarn.lock
> ```
> 
> It is necessary to discuss whether to add a `package-lock` file.

We use pnpm instead, it is pnpm-lock.yaml.

**Comment by suyuan32** [comment]
https://stackoverflow.com/questions/76664013/how-to-use-pnpm-instead-of-npm-in-github-actions

**Comment by suyuan32** [comment]
And you should push to dev branch

**Comment by Jzow** [comment]
Link: 

1. https://github.com/vbenjs/vue-vben-admin/issues/3010
2. https://github.com/vitejs/vite/issues/14350

**Comment by suyuan32** [comment]
Too many commits, you should squash them into 1 or 2 commits. When you finish all the modification, leave a comment.

**Comment by Jzow** [comment]
Fixed, this PR has done two things:

1. We have modified some `less` file naming conventions under the `design module` while also modifying the names referenced by` vue component` files. The purpose of this is that we should not follow the practice of non-standard `vben-ui` naming. We can make appropriate adjustments to facilitate future extensions of new `css` and `less` files that also need to comply with the regulations.
2. Add CI/CD for Github Action.

**Review by suyuan32 — COMMENTED** [review]
ok

**Pull request merged**